index.html

```diff
@@ -1592,17 +1590,40 @@ <h3 id="applying">
         <div class="issue" data-number="446"></div>
       </section>
     </section>
-    <section>
+    <section data-dfn-for="webappmanifest" data-link-for="webappmanifest">
```

Comment:
The `data-dfn-for="webappmanifest" data-link-for="webappmanifest"` need to match actual IDL things, so, should be "WebAppManifest".

Reply:
ok, fixed